### PR TITLE
Add chain 8ball

### PR DESCRIPTION
**Pull request by godev111222333**

### CODE REVIEW

File: 8ball/chain.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    },
+
+    "genesis": {
+      "genesis_url": "https://8ball.info/8ball-genesis.json"
+    },
+    "versions": [
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://8ball.info/8ball.png",
```
Comment (JeremyParish69) [inline 8ball/chain.json:52]: these URIs must point to this repo:
e.g., `https://raw.githubusercontent.com/cosmos/chain-registry/master/8ball/images/8ball.png`

File: 8ball/chain.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "8ball",
+  "status": "live",
+  "website": "https://8ball.info/",
+  "network_type": "mainnet",
+  "pretty_name": "8ball",
+  "chain_id": "eightball-1",
+  "bech32_prefix": "8ball",
```
Comment (JeremyParish69) [inline 8ball/chain.json:9]: Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (godev111222333) [reply 8ball/chain.json:9]: Registered

File: 8ball/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            "denom": "ebl",
+            "exponent": 6
+          }
+        ],
+        "base": "uebl",
+        "name": "8ball",
+        "display": "8ball",
+        "symbol": "EBL",
+        "logo_URIs": {
+          "png": "https://8ball.info/8ball.png",
```
Comment (JeremyParish69) [inline 8ball/assetlist.json:22]: oh, these URIs need the same fix (must point to this repo)

Comment (godev111222333) [reply 8ball/assetlist.json]: Fixed. Thank you

**Pull request merged**